.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
+# Editor-based HTTP Client requests
+/httpRequests/
```

Comment:
Not exactly sure what this `.idea` folder is?  Can the entire folder be added to the main level `.gitignore` or does it serve some function?

Reply:
Removed the `.idea` folder and added it to `.gitignore` in case it pops up again.